### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by nvda-mesharma**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -172,3 +172,5 @@ cython_debug/
 generated_input_data.json
 profile_*.json
 model.cache
+
+
```
Comment (rmccorm4) [inline .gitignore:176]: What's the purpose of this change?

Comment (nvda-mesharma) [reply .gitignore:176]: to trigger the CI :) It will be reverted once everything works as expected
